nilearn/plotting/surf_plotting.py

```diff
@@ -0,0 +1,227 @@
+"""
+Functions for surface visualization.
+Only matplotlib is required.
+"""
+
+from nilearn._utils.compat import _basestring
+from .img_plotting import _get_colorbar_and_data_ranges
+
+# Import libraries
+import numpy as np
+import nibabel
+from nibabel import gifti
+import matplotlib.pyplot as plt
+import matplotlib.tri as tri
+from mpl_toolkits.mplot3d import Axes3D
+
+
+# function to figure out datatype and load data
+def check_surf_data(surf_data, gii_darray=0):
+    # if the input is a filename, load it
+    if isinstance(surf_data, _basestring):
+        if (surf_data.endswith('nii') or surf_data.endswith('nii.gz') or
+                surf_data.endswith('mgz')):
+            data = np.squeeze(nibabel.load(surf_data).get_data())
+        elif (surf_data.endswith('curv') or surf_data.endswith('sulc') or
+                surf_data.endswith('thickness')):
+            data = nibabel.freesurfer.io.read_morph_data(surf_data)
+        elif surf_data.endswith('annot'):
+            data = nibabel.freesurfer.io.read_annot(surf_data)[0]
+        elif surf_data.endswith('label'):
+            data = nibabel.freesurfer.io.read_label(surf_data)
+        elif surf_data.endswith('gii'):
+            data = gifti.read(surf_data).darrays[gii_darray].data
+        else:
+            raise ValueError('Format of data file not recognized.')
+    return data
+
+
+# function to figure out datatype and load data
+def check_surf_mesh(surf_mesh):
+    # if input is a filename, try to load it
+    if isinstance(surf_mesh, _basestring):
+        if (surf_mesh.endswith('orig') or surf_mesh.endswith('pial') or
+                surf_mesh.endswith('white') or surf_mesh.endswith('sphere') or
+                surf_mesh.endswith('inflated')):
+            coords, faces = nibabel.freesurfer.io.read_geometry(surf_mesh)
+        elif surf_mesh.endswith('gii'):
+            coords, faces = gifti.read(surf_mesh).getArraysFromIntent(1008)[0].data, \
```

Comment:
You can use `nibabel.nifti1.intent_codes['NIFTI_INTENT_POINTSET']` rather than `1008`